k8s/src/main/scala/io/buoyant/k8s/SerializationModule.scala

```diff
@@ -0,0 +1,10 @@
+package io.buoyant.k8s
+
+import com.fasterxml.jackson.databind.module.SimpleModule
+
+/**
+ * Class used to provide LoadService-compatible serialization/deserialization plugins to k8s JSON parsing.
+ */
+abstract class SerializationModule {
```

Comment:
What are the benefits of this being pluggable?  Don't the concrete ThirdParty types require specific serialization support?